set.mm

```diff
@@ -662596,6 +663406,1223 @@ closed under the multiplication ( ' X ' ) of a finite number of
+       < ( limsup ` F ) ) $=
+      ( vx cle wbr cxr wcel adantr cr wa cv cfv c2 cdiv cmin clsp cuz wral wrex
+      co clt cres crn csup cmpt cinf cxad nfv uzn0d wss rnresss a1i frexr sstrd
+      frnd supxrcld nfcv limsupreuz mpbid simpld rexr ad4antlr ad2antrr adantll
+      wf uztrn2 ffvelrnd rexrd 3impa ad5ant134 ad4antr simpr wceq eqcomd adantl
+      fvres wfn ffnd fnssres syl2anc fnfvelrnd eqeltrd eqid supxrubd rexlimdva2
+      ssd xrletrd ralimdva reximdva mpd rphalfcld infrpgernmpt w3a simp3 oveq1d
+      limsupvaluz 3ad2ant1 breqtrd caddc 3adantl3 simpl1 syl cvv fexd limsupcld
+      fvexi rpred rehalfcld xaddcld simpl3 rexaddd wb lesubaddd mpbird syld3an3
+      ralrimiva 3exp reximdai wi simpll ffvelrnda resubcld rphalfltd ltletrd ex
+      ltsub2dd ) ACUAZDUBZFUCUDUJZUEUJZDUFUBZNOZCBUAZUGUBZUHZBGUIZYRFUEUJZUUAUK
+      OZCUUDUHZBGUIADUUDULZUMZPUKUNZBGUULUOUMPUKUPZYSUQUJZNOZBGUIUUFABMGUULYSAB
+      URZAEGHIUSAUULPQZUUCGQZAUUKAUUKDUMZPUUKUUSUTADUUDVAVBAGPDAGDJVCZVEVDZVFZR
+      AMUAZYRNOZCUUDUIZBGUHZMSUIZUVCUULNOZBGUHZMSUIAUVGYRUVCNOCGUHMSUIZAUUASQZU
+      VGUVJTKAMCBDEGCDVGHIJVHVIVJAUVFUVIMSAUVCSQZTZUVEUVHBGUVMUURTZUVDUVHCUUDUV
+      NYQUUDQZTZUVDTUVCYRUULUVLUVCPQAUURUVOUVDUVCVKVLAUURUVOYRPQZUVLUVDAUURUVOU
+      VQAUURTZUVOTZYRUVSGSYQDAGSDVOUURUVOJVMUURUVOYQGQZAEYQUUCGIVPVNZVQZVRZVSVT
+      AUUQUVLUURUVOUVDUVBWAUVPUVDWBAUURUVOYRUULNOZUVLUVDAUURUVOUWDUVSUUKYRUULAU
+      UKPUTUURUVOUVAVMUVSYRYQUUJUBZUUKUVOYRUWEWCUVRUVOUWEYRYQUUDDWFWDWEUVSUUDYQ
+      UUJUVRUUJUUDWGZUVOUVRDGWGZUUDGUTUWFAUWGUURAGSDJWHRUVRCUUDGUWAWPGUUDDWIWJR
+      UVRUVOWBWKWLUULWMWNZVSVTWQWOWRWSWTAFLXAXBAUUOUUEBGUUPAUURUUOUUEAUURUUOUUL
+      UUAYSUQUJZNOZUUEAUURUUOXCUULUUNUWINAUURUUOXDAUURUUNUWIWCUUOAUUMUUAYSUQAUU
+      AUUMABDEGHIUUTXFWDXEXGXHAUURUWJXCZUUBCUUDUWKUVOTZUUBYRUUAYSXIUJZNOZUWLYRU
+      WIUWMNUWLYRUULUWIAUURUVOUVQUWJUWCXJUWLAUUQAUURUWJUVOXKZUVBXLUWLAUWIPQUWOA
+      UUAYSADXMAGSXMDJGXMQAGEUGIXPVBXNXOAYSAFAFLXQZXRZVRXSXLAUURUVOUWDUWJUWHXJA
+      UURUWJUVOXTWQUWLAUWIUWMWCUWOAUUAYSKUWQYAXLXHAUURUVOUUBUWNYBUWJUVSYRYSUUAU
+      WBAYSSQZUURUVOUWQVMAUVKUURUVOKVMYCXJYDYFYEYGYHWTAUUEUUIBGUVRUUBUUHCUUDUVS
+      AUVTUUBUUHYIAUURUVOYJUWAAUVTTZUUBUUHUWSUUBTUUGYTUUAUWSUUGSQUUBUWSYRFAGSYQ
+      DJYKZAFSQUVTUWPRZYLRUWSYTSQUUBUWSYRYSUWTAUWRUVTUWQRZYLRAUVKUVTUUBKVMUWSUU
+      GYTUKOUUBUWSYSFYRUXBUXAUWTAYSFUKOUVTAFLYMRYPRUWSUUBWBYNYOWJWRWSWT $.
+  $}
+
+  ${
+    $d F i j l $.  $d X i j k l $.  $d Z i j l $.  $d i l ph $.
+    limsupgt.k $e |- F/_ k F $.
+    limsupgt.m $e |- ( ph -> M e. ZZ ) $.
+    limsupgt.z $e |- Z = ( ZZ>= ` M ) $.
+    limsupgt.f $e |- ( ph -> F : Z --> RR ) $.
+    limsupgt.r $e |- ( ph -> ( limsup ` F ) e. RR ) $.
+    limsupgt.x $e |- ( ph -> X e. RR+ ) $.
+    $( Given a sequence of real numbers, there exists an upper part of the
+       sequence that's appxoximated from below by the superior limit.
+       (Contributed by Glauco Siliprandi, 2-Jan-2022.) $)
```

Comment:
appxoximated --> approximated (same above)
I would write something like: "For any positive real, the superior limit of F is larger than any of its values at large enough arguments, up to that positive real."

Reply:
New commit:
changed as suggested